docs/MigrationGuides/migration_11_to_12.md

```diff
@@ -0,0 +1,17 @@
+# Migration Guide from 11.x to 12.x
+
+This project follows [Semantic Versioning](http://semver.org).
+
+### ReactiveSwift Migration
+- Check the [ReactiveSwift 4.0.0 release notes](https://github.com/ReactiveCocoa/ReactiveSwift/releases/tag/4.0.0) for changes related to ReactiveSwift.
+
+### Result Migration
+- Check the [Result 4.0.0 release notes](https://github.com/antitypical/Result/releases/tag/4.0.0) for changes related to Result.
+
+### AccessTokenPlugin Migration
+- Handle `.custom(String)` case for `AuthorizationType` or add a default case to achive exhaustiveness.
```

Comment:
Nit: `Handle the*`

Reply:
Also, can we explain what this `.custom(String)` does / can be used for?

Reply:
Not sure if it would be worth mentioning that handling the case is the preferred way to exhaust a switch here, as a default case might then not catch any new cases in future migrations.

Reply:
Sounds good to me, I just removed that part relating to the `default` case